From fb5cfd58e6ffe97dacf99f64b30d6ede4387a82e Mon Sep 17 00:00:00 2001
From: Ioana Pandele <ioanap@chromium.org>
Date: Wed, 15 Jun 2022 21:35:52 +0000
Subject: [PATCH 17/59] [Sheriff] Disable BluetoothTest.ConstructDefaultAdapter
 on Mac

(cherry picked from commit 6ab5afb8214f18b886e3b190045ee8a90c3ca5f7)

Bug: 1334462
Change-Id: I95f9678936fa652c75949a48cfc53e788fc4b4a2
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3695178
Auto-Submit: Ioana Pandele <ioanap@chromium.org>
Owners-Override: Ioana Pandele <ioanap@chromium.org>
Commit-Queue: Lan Wei <lanwei@chromium.org>
Reviewed-by: Lan Wei <lanwei@chromium.org>
Owners-Override: Lan Wei <lanwei@chromium.org>
Cr-Original-Commit-Position: refs/heads/main@{#1011954}
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3708698
Commit-Queue: Phillis Tang <phillis@chromium.org>
Owners-Override: Phillis Tang <phillis@chromium.org>
Auto-Submit: Steven Holte <holte@chromium.org>
Cr-Commit-Position: refs/branch-heads/5005@{#1176}
Cr-Branched-From: 5b4d9450fee01f821b6400e947b3839727643a71-refs/heads/main@{#992738}
---
 device/bluetooth/bluetooth_adapter_unittest.cc | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/device/bluetooth/bluetooth_adapter_unittest.cc b/device/bluetooth/bluetooth_adapter_unittest.cc
index cfd96a78408b..8719f0f732b2 100644
--- a/device/bluetooth/bluetooth_adapter_unittest.cc
+++ b/device/bluetooth/bluetooth_adapter_unittest.cc
@@ -708,7 +708,8 @@ TEST_F(BluetoothAdapterTest, StartDiscoverySessionError_Destroy) {
 }
 
 // TODO(scheib): Enable BluetoothTest fixture tests on all platforms.
-#if BUILDFLAG(IS_ANDROID) || BUILDFLAG(IS_MAC)
+// Flaky on Mac. See crbug.com/1334462
+#if BUILDFLAG(IS_ANDROID)
 #define MAYBE_ConstructDefaultAdapter ConstructDefaultAdapter
 #else
 #define MAYBE_ConstructDefaultAdapter DISABLED_ConstructDefaultAdapter
-- 
2.37.0

